docs/architecture/HestiaWeb_architecture.md

```diff
@@ -18,126 +16,101 @@
 
-The clients have created this system using the REST API. This decouples our work from the underlying infrastructure of Hestia.
+Additionally, the client would like to have a *marketplace*, where independed creators can submit plugins to be made available to other Hestia users.
 
+The clients have created this system using the REST API. This decouples our work from the underlying infrastructure of Hestia.
 ## General Overview of the System
 The Hestia Web Interface can be divided into two main sub-systems as mentioned in the introduction.
-
 * Front-end user interface, with which the user interacts
 * Back-end server that connects users to their local Hestia controllers and holds user and server information
-
 ![Website Design](images/diagram.png  "Structure of the system")
-
 ## Website Front-End
 Since the frontend of the website is the first aspect of the product that the customer will interact with, and will be one of the main sources of content or discontent throughout their user experience, a variety of design choices have to be made. 
-
 The website will allow a user to connect to their controllers through a web server, and then manage the devices on their controllers through a range of different controls, and do this in the knowledge that themselves and their information are being kept safe.
-
 ![Website Design](images/Hestia_login_concept.png  "Website Login Concept")
-
 ![Website Design](images/Hestia_dash_concept.png  "Website Dash Concept")
-
 ![Website Design](images/Hestia_control_concept.png  "Website Control Concept")
-
 ### Design choices
-Due to the Firebase server, once the user has logged in then they will be able to see devices from every server they own, without having to switch between them. Thus, the main focus of the website will be on listing the devices managed by their controllers, and on the operations that can be applied on them. These will mirror those already implemented by Hestia, such as renaming or deleting a device, but with more streamlined interfacing added, such as using buttons and sliders in order to change the activators of a device, instead of having to directly enter values.
-
-
+We use Auth0 for login, in order to implement single sign-on, and once the user is logged in the user's information will be used to get all their servers from our PostgreSQl, and connect to these servers. The frontend will be developed in Vue.js, in  order to simplify the development of our user interface. The main focus of the website will be on listing the devices managed by their controllers, and on the operations that can be applied on them. These will mirror those already implemented by Hestia, such as renaming or deleting a device, but with more streamlined interfacing added, such as using buttons and sliders in order to change the activators of a device, instead of having to directly enter values.
```

Comment:
Is there a specific reason you chose PostgreSQL and any of the other SQL variants?

Reply:
Mostly since I was in charge of the database, and I currently am working with postgres at work (where I am also making a backend), I chose to keep them the same while learning. The actual server choice won't matter in the future since alchemy can use any. We can speak with the client again to be sure which they want.

Reply:
Fair, indeed communicate with the client about a possible preference for the end product.